src/site/content/en/blog/samesite-cookies-explained/index.md

```diff
@@ -19,6 +19,17 @@ tags:
   - cookies
 ---
 
+If you're just here for the `SameSite` cookie guidance, you can skip to the
```

Comment:
Won't most folks be here for the SameSite cookie guidance...?

Reply:
Hopefully addressed in the changes!